compose/backend.yml

```diff
@@ -3,7 +3,7 @@ version: '3'
 
 services:
   ccd-user-profile-api:
-    image: "hmcts/ccd-user-profile-api:${CCD_USER_PROFILE_API_TAG:-latest}"
+    image: "hmcts.azurecr.io/hmcts/ccd-user-profile-api:${CCD_USER_PROFILE_API_TAG:-latest}"
```

Comment:
@dwaynebailey I've tested this fix with the mechanism of switching branches. It seems to work with both remote and local branches. @veeresha-hmcts please double check this fix does not break:
https://github.com/hmcts/ccd-docker#compose-branches